Good-Bids/goodbids
# [#408] Updating UI of site directory

# Summary

This updates the site directory by:

- Filter out the network site
- Update grid to 5
- Pull in the NP site logo (if we have one)
- Update the link text display

## Issues

* #408

## Testing Instructions

1. Go to any page in the main site and add the Site Directory block. 
2. Make sure it matches four new updates listed. 

## Screenshots

<img width="1338" alt="Screenshot 2024-02-15 at 1 12 48 PM" src="https://github.com/Good-Bids/goodbids/assets/91974372/af545c4c-d075-4d21-b79d-3a8566caa07a">
